packages/test/harness/src/host/common/allImagesCompleted.js

```diff
@@ -0,0 +1,16 @@
+const until = require('./until');
```

Comment:
Just confirming, this runs automatically on every test, right?

Reply:
This run before every snapshot is taken. 🎉